Engine/util/library_posix.h

```diff
@@ -48,8 +48,12 @@ class PosixLibrary : BaseLibrary
 
   AGS::Common::String BuildPath(char *path, AGS::Common::String libraryName)
   {
-    AGS::Common::String platformLibraryName = path;
-    platformLibraryName.Append("/lib");
+    AGS::Common::String platformLibraryName = "";
+    if (path) {
+      platformLibraryName = path;
```

Comment:
The code style for our project suggests to put curvy braces on separate lines (you may refer to how this is done in the rest of the file).